apps/storybook/src/NotificationMarker.stories.tsx

```diff
@@ -0,0 +1,134 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import {
+  Text,
+  IconButton,
+  NotificationMarker,
+  NotificationMarkerProps,
+} from '@itwin/itwinui-react';
+import SvgNotification from '@itwin/itwinui-icons-react/cjs/icons/Notification';
+
+export default {
+  component: NotificationMarker,
+  args: {
+    enabled: true,
+    pulsing: false,
+  },
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    status: {
+      control: 'radio',
+      options: ['primary', 'positive', 'warning', 'negative'],
+    },
+  },
+  title: 'Core/NotificationMarker',
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'flex-start',
+          gap: '20px',
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+} as Meta<NotificationMarkerProps>;
+
+export const Basic: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <>
+      <IconButton styleType='borderless'>
+        <NotificationMarker {...args}>{args.children}</NotificationMarker>
+      </IconButton>
+      <NotificationMarker {...args}>3 members online</NotificationMarker>
+    </>
+  );
+};
+
+Basic.args = {
+  status: 'primary',
+  children: <SvgNotification />,
+};
+
+Basic.argTypes = {
+  children: {
+    control: false,
+  },
+};
+
+export const Pulsing: Story<NotificationMarkerProps> = (args) => {
+  return (
+    <>
+      <IconButton styleType='borderless'>
+        <NotificationMarker status='primary' {...args}>
+          <SvgNotification />
+        </NotificationMarker>
+      </IconButton>
+
+      <NotificationMarker status='primary' {...args}>
+        <Text style={{ fontWeight: '700' }}>URGENT</Text>
```

Comment:
How to add bold text using our `<Text>` component? Is this the way?

Reply:
In `ComboBox.stories.tsx` the fontWeight is set to 'bold', but it looks like it's the same thing (ie 'bold' == '700'), so I think it's fine as is. Doesn't seem to be another way to do it.
![image](https://user-images.githubusercontent.com/37936169/208146462-3407da14-7212-4127-adb1-54f876a8a68d.png)